### ISSUE TITLE
'selectionChange' event emitted twice, when selecting item that it has empty value.

### ISSUE THREAD

**Issue by tkmknd** [angular/material2#13211]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
'selectionChange' event always emitted only once, when changing value.

#### What is the current behavior?
'selectionChange' event emitted twice, when selecting item that it has empty value.

e.g.
`<mat-select (selectionChange)="changeEvent()">`
`<mat-option>sample</mat-option>`
`</mat-select>`

#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>

#### What is the use-case or motivation for changing an existing behavior?
Since event emitted twice, event function (e.g. : changeEvent()) is executed twice.
If I would't like to execute event function twice, I would have to implement check code to prevent  function from executing twice.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular:6.1.4
Angular Material:6.4.6
OS:Any
TypeScript:2.7.2
Browser:Any

**Comment by crisbeto** [comment]
Closing as a duplicate of https://github.com/angular/material2/issues/10675.
